commit c8df52eccfa2d1ace13d887aa6bdab1178c1edb4
Author: David S. Miller <davem@davemloft.net>
Date:   Tue Dec 4 16:05:57 2012 -0500

    Fix sparc build with older compilers.
    
    	* sysdeps/generic/memcopy.h: Add multiple inclusion protection.

diff --git a/sysdeps/generic/memcopy.h b/sysdeps/generic/memcopy.h
index a8e5a44..631214b 100644
--- a/sysdeps/generic/memcopy.h
+++ b/sysdeps/generic/memcopy.h
@@ -1,5 +1,5 @@
 /* memcopy.h -- definitions for memory copy functions.  Generic C version.
-   Copyright (C) 1991, 1992, 1993, 1997, 2004 Free Software Foundation, Inc.
+   Copyright (C) 1991, 1992, 1993, 1997, 2004, 2012 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Torbjorn Granlund (tege@sics.se).
 
@@ -17,6 +17,9 @@
    License along with the GNU C Library; if not, see
    <http://www.gnu.org/licenses/>.  */
 
+#ifndef _MEMCOPY_H
+#define _MEMCOPY_H	1
+
 /* The strategy of the memory functions is:
 
      1. Copy bytes until the destination pointer is aligned.
@@ -144,3 +147,5 @@ extern void _wordcopy_bwd_dest_aligned (long int, long int, size_t) __THROW;
 
 /* Threshold value for when to enter the unrolled loops.  */
 #define	OP_T_THRES	16
+
+#endif /* memcopy.h */
